client/router.js

```diff
@@ -206,6 +206,8 @@ ClientRouter.prototype.redirectTo = function(path, options) {
 
   if (options.pushState === false) {
     // Do a full-page redirect.
+    // add a leading slash so path is not appended
+    if (path.charAt(0) !== '/') path = '/' + path;
```

Comment:
What's the result of this if `pushState === true`?  Also, mind adding a quick test to this to make sure it works in both of those cases?